src/txn/txn_ckpt.c

```diff
@@ -2101,13 +2101,10 @@ __wt_checkpoint_close(WT_SESSION_IMPL *session, bool final)
         return (__wt_evict_file(session, WT_SYNC_DISCARD));
 
     /*
-     * Don't flush data from modified trees independent of system-wide checkpoint when either there
-     * is a stable timestamp set or the connection is configured to disallow such operation.
-     * Flushing trees can lead to files that are inconsistent on disk after a crash.
+     * Don't flush data from modified trees independent of system-wide checkpoint. Flushing trees
+     * can lead to files that are inconsistent on disk after a crash.
      */
-    if (btree->modified && !bulk && !__wt_btree_immediately_durable(session) &&
-      (S2C(session)->txn_global.has_stable_timestamp ||
-        (!F_ISSET(S2C(session), WT_CONN_FILE_CLOSE_SYNC) && !metadata)))
+    if (btree->modified && !bulk && !__wt_btree_immediately_durable(session) && !metadata)
```

Comment:
I thought this change would remove the `__wt_btree_immediately_durable()` test as well. I think that individually checkpointing logged tables breaks reading from checkpoints (at least), and I don't see any advantage to doing that.

Reply:
@hbokhari said we should keep this change. My opinion is can we keep it for now and fix this when we try to get the checkpoint cursor working?

Reply:
@kommiharibabu, can you please explain why you want to keep the `__wt_btree_immediately_durable()` test, specfiically, why the ability to separately checkpoint log-commit files is a good thing?

Reply:
Is it because the data would be the same no matter you checkpointed or not after restart for logged table as log recovery should always run?

Reply:
The main reason behind allowing logged tables to let them checkpointed individually is they don't have any difference because of logged after restart as @quchenhao said. Regarding supporting checkpoint cursors on these logged tables we may or may not be required to support the individual checkpoints. I also feel it is better to decide during the support of checkpoint cursors whether we allow individual checkpoints for logged tables.

Reply:
@kommiharibabu, @quchenhao: The problem I have is I don't understand why we would WANT to allow logged tables to individually checkpoint. The reason here seems to be more than we CAN allow it, and I agree, but I don't see why we would want to.

It's an edge case we have to document and test, and I can't see any advantage to having an exception to a general rule.

Can someone say why applications might want the ability to (1) individually checkpoint or (2) do exclusive operations on logged tables, while not being able to do so with non-logged tables?

If there's something good here I can see keeping the exception, but right now I can't see the reason we'd want to treat logged and non-logged tables differently.

@quchenhao, speaking of which, we will need some general wording in the documentation about this. I can take a run at that if you'd like, up to you.

Reply:
@keithbostic I'll try to disable the logged tables as well.